### PR TITLE
Feature/8 to create database

### PR DESCRIPTION
**Pull request by TREN-D**

### CODE REVIEW

File: InternetHospital.WebApi/Program.cs [hunk 1]
```diff
@@ -3,18 +3,48 @@
...
+            var host = CreateWebHostBuilder(args).Build();
+
+
+            using (var scope = host.Services.CreateScope())
+            {
+                var services = scope.ServiceProvider;
+                try
+                {
+
+                    var userManager = services.GetRequiredService<UserManager<User>>();
```
Comment (voznovych) [inline InternetHospital.WebApi/Program.cs]: Delete userManager

**Review by voznovych — COMMENTED** [review]
Update to code convention.

File: InternetHospital.WebApi/Program.cs (hunk 1 above)
Comment (TREN-D) [reply InternetHospital.WebApi/Program.cs]: Deleted

**Review by voznovych — APPROVED** [review]
Replace initialization of statuses by consts with initialization by json.

**Review by andriipetryna — APPROVED** [review]

**Review by serhiidotsyk — APPROVED** [review]

**Pull request merged**